documentation/asciidoc/topics/proc_disabling_fips_mode_cr.adoc

```diff
@@ -0,0 +1,33 @@
+[id="disabling-fips-mode-CR_{context}"]
+= Disabling FIPS mode in your `Infinispan` CR
+
+The {openshiftplatform} can use certain Federal Information Processing Standards (FIPS) components that ensure {openshiftshort} clusters meet the requirements of a FIPS compliance audit. 
+This might cause issues when you want your {brandname} instance to run on any {openshiftshort} cluster that has FIPS mode enabled. 
+{brandname} {ProductVersion} does not support FIPS mode, so you must disable FIPS mode in your `Infinispan` CR.
+
+After you disable FIPS mode in your `Infinispan` CR, any component that uses a JVM, such as {brandname} Server, {brandname} CLI, or Gossip router, ignores FIPS mode. 
+This happens, because the JVM no longer loads FIPS-related cryptographic libraries on implementation startup. 
+
+[NOTE]
+====
+{openshiftshort} might disable FIPS mode by default.
```

Comment:
This is redundant. It definitely is disabled by default.

Reply:
Removed.